geniza/corpus/admin.py

```diff
@@ -343,21 +345,33 @@ def tabulate_queryset(self, queryset):
                     "pgpid": doc.id,
                     "url": absolutize_url(doc.get_absolute_url()),
                     "iiif_urls": ";".join(
-                        [fragment.iiif_url for fragment in all_fragments]
+                        [
+                            fragment.iiif_url
+                            for fragment in all_fragments
+                            if fragment.iiif_url
```

Comment:
I don't think this is the logic we want (here and throughout), because if one fragment has a value and others don't we won't know which one it belongs to (unless we think they don't care about that?).

My recommendation would be to only generate the field if any fragments have the value (i.e. `any(fragment.iiif_url for fragment in all_fragments)`) but not to skip empty values.

Reply:
I wrapped them in a lambda function:
```python
semicolon_str = lambda x: "" if x == ";" else x
```
It seemed simpler to me, but I recognize it's atypical. Is that alright? (e7778bdc35a5bb02a24675081da64b117845fa35)

Reply:
seems ok. Please add a brief comment above the lambda so we remember what this is when we come back to it later 🙂